dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -117,6 +117,97 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId) {
   }
 };
 
+function shareProject() {
```

Comment:
Yep!  The only think I changed was `shareUrl` / `encodedShareUrl` to work on non-project pages.